game/src/router/index.ts

```diff
@@ -0,0 +1,23 @@
+import { createRouter, createWebHistory } from 'vue-router'
+import game from '../views/game.vue'
+
+const router = createRouter({
+  history: createWebHistory(import.meta.env.BASE_URL),
+  routes: [
+    {
+      path: '/',
+      name: 'home',
+      component: game
+    },
+    {
+      path: '/about',
+      name: 'about',
+      // route level code-splitting
+      // this generates a separate chunk (About.[hash].js) for this route
+      // which is lazy-loaded when the route is visited.
+      component: () => import('../views/AboutView.vue')
+    }
+  ]
+})
```

Comment:
Create a simple page called game.vue ✅

Page should have a card with a heading that with a clever name for the game ✅
some default placeholder body text ✅
and a button to take you back to the home page ❌
there should be a button on the home page to access this page ❌

Right now your game is being rendered on the home route. The instructions want you to have a game that contains a button that will take you back to the home page. My suggestion would be to create another route called /game or something along those lines and then add a button in that view that will redirect the user back to the home view when clicked.

Here is a link to our site for reference: https://blue-mushroom-0022f9f1e.3.azurestaticapps.net/